packages/synapse-react-client/src/components/ConfirmationDialog/ConfirmationDialog.tsx

```diff
@@ -15,6 +15,14 @@ export type ConfirmationButtonsProps = {
 
 export const CANCEL_BUTTON_TEXT = 'Cancel'
 
+export function CancelButton(props: { onCancel: () => void }) {
```

Comment:
I was going to extract and reuse this then decided against it, kept the extraction